src/components/App/App.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from "react";
+import TodoList from "../TodoList/TodoList";
+import AppHeader from "../AppHeader/AppHeader";
+import SearchPanel from "../SearchPanel/SearchPanel";
+import "./App.scss";
+import ItemStatusFilter from "../ItemStatusFilter/ItemStatusFilter";
+import ItemAddForm from "../ItemAddForm/ItemAddForm";
+
+export default class App extends Component {
+  maxId = 100;
```

Comment:
@Ihor-Onyshchuk  to get a unique id use approach like this `const uuid = () => `uuid${Date.now().toString(16)}`;`
and then you can call `uuid()` where you need